holidays/groups/mandaean.py

```diff
@@ -0,0 +1,107 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from datetime import date
+from typing import Optional
+
+from holidays.calendars.gregorian import _timedelta
+from holidays.calendars.mandaean import _Mandaean
+
+
+class MandaeanHolidays:
+    """
+    Mandaean holidays.
+    """
+
+    def __init__(self) -> None:
+        self._mandaean = _Mandaean()
+
+    def _add_mandaean_holiday(self, name: str, dt: Optional[date]) -> Optional[date]:
+        """
+        Add a Mandaean holiday.
+        """
+        if dt is None:
+            return None
+        return self._add_holiday(name, dt)
+
+    @property
+    def _parwanaya(self):
+        return self._mandaean.mandaean_to_gregorian(self._year - 1, 13, 1)
+
+    @property
+    def _dehwa_hanina(self):
+        return self._mandaean.mandaean_to_gregorian(self._year - 1, 9, 23)
+
+    @property
+    def _great_feast(self):
+        return self._mandaean.mandaean_to_gregorian(self._year, 1, 6)
```

Comment:
It's better to place properties rigth after `__init__,` and in alphabetical order.